man/BinghamTrib.Rd

```diff
@@ -1,74 +1,60 @@
+% Generated by roxygen2: do not edit by hand
```

Comment:
It's pretty difficult to see changes to the generated Rd because of changes in format and order of sections. Do you think it's safe to assume nothing was lost?

Reply:
I think roxygen2 has is preferred way of creating .Rd files (and I'm not sure how much of it can be tweaked, if documentation is meant to be uniform across packages). I don't think any data has been lost, rather, roxygen2 has just shuffled some of the sections around and removed the indentation that was previously in the .Rd files.